src/deepsparse/pipeline.py

```diff
@@ -257,7 +257,7 @@ def __call__(self, *args, **kwargs) -> BaseModel:
         # join together the batches of size `self._batch_size`
         engine_outputs = self.join_engine_outputs(batch_outputs)
         timer.stop(InferencePhases.ENGINE_FORWARD)
-
+        # let's put this into a separate metric group!
         self.log(
             identifier=f"{REQUEST_DETAILS_IDENTIFIER_PREFIX}/input_batch_size",
```

Comment:
Let's not log this value under "request details" -> let's uncouple it from the server context. Maybe we can have something like `pipeline properties`?